include/task.h

```diff
@@ -1,6 +1,10 @@
 #ifndef GUARD_TASK_H
 #define GUARD_TASK_H
 
+// general task defines
+#define TASK gTasks[taskId]
+#define FUNC func
```

Comment:
yes, but since multiple files use local task defines, if I don't put them in a more global header, it's going to cause console errors (albeit they wont cause the build nor Travis to fail) because its declared in multiple places.